swift/apple/README.md

```diff
@@ -27,11 +27,11 @@ CI/CD pipeline.
    cd swift/apple
    ```
 
-1. Rename and populate xcconfig files:
+1. Copy an appropriate xcconfig and edit as necessary:
 
    ```bash
-   cp Firezone/xcconfig/Developer.xcconfig.firezone Firezone/xcconfig/Developer.xcconfig
-   vim Firezone/xcconfig/Developer.xcconfig
+   cp Firezone/xcconfig/dev.xcconfig Firezone/xcconfig/config.xcconfig
+   vim Firezone/xcconfig/config.xcconfig
    ```
 
    ```bash
```

Comment:
I think we should remove the lines about Server.xcconfig from the README.

Reply:
Good catch, also updated our constants error message text to mention the new filename.